_includes/pbc/all/install-features/202304.0/install-the-picker-user-login-feature.md

```diff
@@ -0,0 +1,286 @@
+
+```php
+<?php
+
+namespace Pyz\Zed\Installer;
+
+use Spryker\Zed\Installer\InstallerDependencyProvider as SprykerInstallerDependencyProvider;
+use Spryker\Zed\OauthWarehouse\Communication\Plugin\Installer\OauthWarehouseInstallerPlugin;
+
+class InstallerDependencyProvider extends SprykerInstallerDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\Installer\Dependency\Plugin\InstallerPluginInterface|\Spryker\Zed\InstallerExtension\Dependency\Plugin\InstallerPluginInterface>
+     */
+    public function getInstallerPlugins(): array
+    {
+        return [
+            new OauthWarehouseInstallerPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/Oauth/OauthDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Installer;
+
+use Spryker\Zed\Installer\InstallerDependencyProvider as SprykerInstallerDependencyProvider;
+use Spryker\Zed\OauthWarehouse\Communication\Plugin\Installer\OauthWarehouseInstallerPlugin;
+
+class InstallerDependencyProvider extends SprykerInstallerDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\Installer\Dependency\Plugin\InstallerPluginInterface|\Spryker\Zed\InstallerExtension\Dependency\Plugin\InstallerPluginInterface>
+     */
+    public function getInstallerPlugins(): array
+    {
+        return [
+            new OauthWarehouseInstallerPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/Oauth/OauthDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Oauth;
+
+use Spryker\Zed\Oauth\OauthDependencyProvider as SprykerOauthDependencyProvider;
+use Spryker\Zed\OauthWarehouse\Communication\Plugin\Oauth\WarehouseOauthRequestGrantTypeConfigurationProviderPlugin;
+use Spryker\Zed\OauthWarehouse\Communication\Plugin\Oauth\WarehouseOauthScopeProviderPlugin;
+use Spryker\Zed\OauthWarehouse\Communication\Plugin\Oauth\WarehouseOauthUserProviderPlugin;
+
+class OauthDependencyProvider extends SprykerOauthDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\OauthExtension\Dependency\Plugin\OauthUserProviderPluginInterface>
+     */
+    protected function getUserProviderPlugins(): array
+    {
+        return [
+            new WarehouseOauthUserProviderPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\OauthExtension\Dependency\Plugin\OauthScopeProviderPluginInterface>
+     */
+    protected function getScopeProviderPlugins(): array
+    {
+        return [
+            new WarehouseOauthScopeProviderPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\OauthExtension\Dependency\Plugin\OauthRequestGrantTypeConfigurationProviderPluginInterface>
+     */
+    protected function getOauthRequestGrantTypeConfigurationProviderPlugins(): array
+    {
+        return [
+            new WarehouseOauthRequestGrantTypeConfigurationProviderPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+1. Make sure that you can send the following requests:
```

Comment:
* We have to extend the verification with some generic steps that include:
* The customer has to pick the user and make him the warehouse user then assign him to the warehouse.
* It's also good to include the request body example that can be used as a reference.